cdk/lib/gatehouse.ts

```diff
@@ -37,12 +63,66 @@ export class Gatehouse extends GuStack {
             app: ec2App,
             instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
             access: {scope: AccessScope.PUBLIC},
-            userData: {
-                distributable: {
-                    fileName: `${ec2App}.deb`,
-                    executionStatement: `dpkg -i /${ec2App}/${ec2App}.deb`,
-                },
-            },
+            userData: [
+                '#!/bin/bash -ev',
+
+                // See https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/linux-rpm-demo.md
+                '# Install X-Ray Collector',
+                'wget -P /tmp https://aws-otel-collector.s3.amazonaws.com/ubuntu/arm64/latest/aws-otel-collector.deb',
```

Comment:
in the future we might want to consider not failing the deployment if something in the x-ray setup fails